ote/src/cljs/ote/ui/form_fields.cljs

```diff
@@ -1,6 +1,8 @@
 (ns ote.ui.form-fields
   "UI components for different form fields."
-  (:require [reagent.core :as r]
+  (:require-macros [cljs.core.async.macros :refer [go go-loop]])
```

Comment:
Remove unused core.async stuff on lines 3 and 4.

Reply:
Good catch, removed